### ISSUE TITLE
Update the Docs

### ISSUE THREAD

**Issue by ClemensElflein** [javalin/javalin-vite#4]
Everything is much simpler now, we need to update the docs.

**Comment by tipsy** [comment]
You probably want to have an entry here: https://javalin.io/plugins/

**Comment by ClemensElflein** [comment]
Good idea, so we just fork the https://github.com/javalin/javalin.github.io repo and PR the plugin changes as soon as it's done?

Edit: Looks like it, found the contribution guide.

**Comment by tipsy** [comment]
Correct!
